test/integration/targets/iam_user_info/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
```

Comment:
it would be good to move these tests to iam_user and add iam_user_info as an alias.

Reply:
https://github.com/ansible/ansible/pull/23382/commits/f44a16ce370dcfa99846097ce0acefd9afd9bc3a moves these and adds iam_user_info as an alias.